src/openforms/prefill/api/views.py

```diff
@@ -67,3 +84,90 @@ def get_objects(self):
         choices = plugin.get_available_attributes()
 
         return [ChoiceWrapper(choice) for choice in choices]
+
+
+@extend_schema_view(
+    get=extend_schema(summary=_("List available objecttypes for Objects API")),
+    parameters=[OBJECTS_API_GROUP_QUERY_PARAMETER],
+)
+class PluginObjectsAPIObjecttypeListView(ListMixin, APIView):
+    """
+    List the available prefill objecttypes for Objects API plugin.
+    """
+
+    authentication_classes = (authentication.SessionAuthentication,)
+    permission_classes = (permissions.IsAdminUser,)
+    serializer_class = PrefillObjectsAPIObjecttypeSerializer
+
+    def get_objects(self) -> list[dict[str, Any]]:
+        input_serializer = ObjectsAPIGroupInputSerializer(
+            data=self.request.query_params
+        )
+        input_serializer.is_valid(raise_exception=True)
+
+        config_group = input_serializer.validated_data["objects_api_group"]
+
+        with get_objecttypes_client(config_group) as client:
+            return client.list_objecttypes()
+
+
+@extend_schema_view(
+    get=extend_schema(summary=_("List available objecttype versions for Objects API")),
+    parameters=[OBJECTS_API_GROUP_QUERY_PARAMETER],
+)
+class PluginObjectsAPIObjecttypeVersionListView(ListMixin, APIView):
```

Comment:
Yes I can use them as the base class and have the new one here for the different urls as well. I guess you mean I have to do the same for the serializers since some of them are the same with the ones in registrations.